torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1946,9 +1936,9 @@ void ONNXShapeTypeInference(
     Node* n,
     const ParamMap& params_dict,
     int opset_version) {
-  std::unordered_map<std::string, std::string> outputs_map;
-  auto generated_shape_data = ConstantValueMap::GetGeneratedShape();
-  auto symbol_map = ConstantValueMap::GetSymbolMap();
+  std::unordered_map<std::string, std::string> torch_to_onnx_output_name;
+  auto& generated_shape_data = ConstantValueMap::GetGeneratedShapeDataByName();
+  auto& symbol_map = ConstantValueMap::GetSymbolDimMap();
```

Comment:
```suggestion
  auto& symbol_dim_map = ConstantValueMap::GetSymbolDimMap();
```

Reply:
To make it consistent, I changed symbol_map into symbol_dim_map globally in this file.